### DISCUSSION TITLE
New placeholders

### DISCUSSION THREAD

**Discussion by LuluxClient** [BG-Software-LLC/SuperiorSkyblock2#1171] [Pending Feature Requests] [answered]
Hey it would be nice to have a placeholder to see the TOTAL amount of blocks that needs to be broken. Not the blocks. For example:
> &a%superior_oneblock_phase_block% &7/ &a%superior_oneblock_phase_block_total%
returns: 10 / 5000

Another cool placeholder would be to add phases specific placeholders so we could use them in menus for example:
%superior_oneblock_phase_block_<PHASE NAME>% and %superior_oneblock_phase_block_total_<PHASE NAME>%

And some placeholders to get the next phase name and the number of "oneblock blocks" broken. 

Feel free to check this out: https://docs.bentobox.world/en/latest/gamemodes/AOneBlock/ it has a lot of cool placeholders

**Comment by Tptuaazn** [comment]
Why you not use that plugin:)

**Comment by LuluxClient** [comment]
Because I don’t really like it and superior is a lot better but it’s
missing some things

**Comment by OmerBenGera** [comment, marked as answer]
Added :)
